### PR TITLE
feat(sonarqube): Custom action to create a SonarQube project

### PR DESCRIPTION
**Pull request by jlmayorga**
New Backstage [custom action](https://backstage.io/docs/features/software-templates/writing-custom-actions#registering-custom-actions) to create a SonarQube project

Takes care of #177

### CODE REVIEW

File: plugins/sonarqube-actions/src/actions/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export {createSonarQubeProjectAction} from './createSonarQubeProject'
```
Comment (tumido) [inline plugins/sonarqube-actions/src/actions/index.ts:1]: Missing EOL at EOF

File: plugins/sonarqube-actions/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@janus-idp/sonarqube-actions",
```
Comment (tumido) [inline plugins/sonarqube-actions/package.json:2]:
```suggestion
  "name": "@janus-idp/backstage-scaffolder-backend-module-sonarqube",
```

Looking at all available scaffolder plugins out there, all use a more descriptive name that contains `scaffolder-backend-module`. Let's follow the conventions, so the module is easier to discover for users.

**Review by tumido — CHANGES_REQUESTED** [review]
Hey @jlmayorga! Thanks for contributing this! 👍 I have some formalities/nits inline.

File: plugins/sonarqube-actions/src/actions/index.ts (hunk 1 above)
Comment (jlmayorga) [reply plugins/sonarqube-actions/src/actions/index.ts]: Fixed in c0af8ee

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by tumido** [comment]
> Should the plugin directory be renamed as well?

@jlmayorga I don't see a need to rename the folder at this point. We can give the folder structure a redesign/revisit at later point. Right now, I think this is fine, it's understandable for us (me, you, other maintainers). When it becomes a challenge on that front, we may consider renaming it. 🙂

**Pull request merged**